.travis.yml

```diff
@@ -7,8 +7,8 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-23.0.2
-    - android-23
+    - build-tools-26.0.2
+    - android-26
```

Comment:
Is that really a good idea? Wouldn't this mean, that we require API 26 for running the app? I think we should be _very_ conservative in requiring new features and only when really needed, I think excluding users with old devices just for developers convenience or just because there is something new out there is not the good way.

Reply:
It just means that the APK will be built against the Android 8 SDK.
The minimum SDK is 14 (Android 4.0)

Reply:
ah shit, it is only in the .travis.yml I didn't look closely, sorry. I see now, that there is         minSdkVersion 14 in apps build.gradle, where you also adjusted the targetSdkVersion to 26, which seem reasonable (after searching the difference)
Thanks for teaching me!

Reply:
No worries, the travis file just needs to match the `compileSdkVersion` in the `build.gradle` 😄